12272020/update.sh

```diff
@@ -16,6 +16,16 @@ if [ ! -f "/home/ark/.config/.update${RIDGEK_DATE}" ]; then
 		sudo apt update && sudo apt install rclone -y || (printf "\nCould not install required dependencies\n" | tee -a "$LOG_FILE" && exit 1)
 	fi
 
+	# Install pip
+	if ! pip3 --version &> /dev/null; then
+		sudo apt update && sudo apt install python3-pip -y || (printf "\nCould not install required dependencies\n" | tee -a "$LOG_FILE" && exit 1)
```

Comment:
No need to install pip to get pyudev.  We can just do `sudo apt install python3-pyudev`

Reply:
That's great, I had no idea! (Obviously! 😅)

Reply:
How did you create that test case in your video?

Reply:
I want to try out whiptail using the wrapper script and joy2key you setup in your fork.

Reply:
I didn't test the wrapper script at all yet, but I think it should work. So, just set up a file like 
```shell
#!/bin/bash
whiptail \
    --title "Hello, world!" \
    --msgbox "All your base are belong to us." 8 32
```
and then pass it to the wrapper
```shell
$ /opt/joy2key/listen.sh helloWorld.sh
```
or call it manually as in /opt/joy2key/example.sh